tasks/rh-sign-cosign-image/tests/test-rh-sign-cosign-image-multiple-components.yaml

```diff
@@ -0,0 +1,130 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-rh-sign-cosign-image-multiple-components
+spec:
+  description: Test signing a single image with cosign 
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup-values
+            image: quay.io/konflux-ci/release-service-utils:38c3bfd7479c86b832cba5b61f9bbde40c469393
+            script: |
+              #!/usr/bin/env sh
+              set -eux
+              _TEST_MANIFEST_LIST_DIGEST="sha256:1111"
+              _TEST_MANIFEST_LIST_MDIGEST1="sha256:1111-1"
+              _TEST_MANIFEST_LIST_MDIGEST2="sha256:1111-2"
```

Comment:
I don't see this used

Reply:
Fixed in https://github.com/konflux-ci/release-service-catalog/pull/467.